.gitattributes

```diff
@@ -0,0 +1,16 @@
+/.* export-ignore
+/dist/ export-ignore
+/fixtures/ export-ignore
+/requirement-checker/ export-ignore
+/res/ export-ignore
```

Comment:
this directory is necessary, it contains both the config schema and the requirement checker to ship if you're using Box as a composer package